### ISSUE TITLE
BUG: Invoker never Persists project settings.

### ISSUE THREAD

**Issue by sbrow** [beyondcode/invoker-community#196]
It worked for me for a very short time a few versions back, but doesn't seem to do anything anymore.

I am currently on:
Invoker: v2.4.0
OS: mac OS Big Sur 11.3

I do see this error which might be related?

![image](https://user-images.githubusercontent.com/6729162/117352989-9e9a7600-ae7d-11eb-9088-801b48fde2e5.png)

I access all of my projects through ssh, to either a local vm (virtualbox+Homestead) or a remote machine.

**Comment by mechelon** [comment]
Does this issue still exist with the latest version?

**Comment by mechelon** [comment]
I will close this issue due to inactivity.

If this issue still exists, feel free to reopen and, if applicable, provide more details.

**Comment by sbrow** [comment]
Yes! I am currently on version 2.8.0 and I am still having this problem. I tried opening old projects and starting new ones, either way it doesn't save anything in the project dir.

**Comment by mechelon** [comment]
Just to specify this: Do you have problems with the project settings at the start screen, like here:

<img width="667" alt="CleanShot 2021-09-25 at 15 20 53@2x" src="https://user-images.githubusercontent.com/26432041/134773039-3f879609-8c5d-44e2-a261-3fd5e14df4ce.png">


Or are you talking about the .invoker folder with plugins etc.?

**Comment by sbrow** [comment]
I can toggle the option to save project settings or not, but no invoker specific files ever show up in my projects.

**Comment by cberane** [comment]
I can still confirm the behavior: no `.invoker` folder will be created
![image](https://user-images.githubusercontent.com/24572192/190327262-934dc563-3f41-4b4d-bb73-26525555e46c.png)


I tested local project:
![image](https://user-images.githubusercontent.com/24572192/190327374-13354cdc-38e9-4726-8cde-be2271b34758.png)
![image](https://user-images.githubusercontent.com/24572192/190327382-019915b1-9b92-4ca7-948f-419c85f2f0eb.png)

as well as with SSH tunnel into homestead:
![image](https://user-images.githubusercontent.com/24572192/190327429-5fb37076-a83c-45c8-8ecb-8cbd4d5590e4.png)
![image](https://user-images.githubusercontent.com/24572192/190327446-1524beb5-3942-4d37-88ef-0351714b5430.png)
